ethcore/src/ethereum/ethash.rs

```diff
@@ -215,8 +225,9 @@ impl Engine for Arc<Ethash> {
 		} else if block_number < self.ethash_params.eip150_transition {
 			Schedule::new_homestead()
 		} else {
+			let max_code_size = if block_number >= self.ethash_params.eip160_transition { self.ethash_params.max_code_size as usize } else { usize::max_value() };
```

Comment:
Isn't that a breaking change for some private chains? Should we include it in relnotes?

Reply:
True, although eip160 was added and enabled a long time ago. All non-ethash engines have it enabled unconditionally.